test/compression.js

```diff
@@ -631,6 +673,34 @@ describe('compression()', function () {
         .end()
     })
 
+    if (hasBrotliSupport) {
```

Comment:
Instead of excluding the test, have the test always be there but show up as skipped in the mocha output when the support is missing.